src/base.css

```diff
@@ -18,7 +18,8 @@ body {
   border: 2px solid black !important;
 }
 
-#app {
+#app,
```

Comment:
Will we even have a `#app` anymore? If not, should we remove these lines?

Reply:
works for me! I was being conservative initially until everything was moved. Cleaning now